package.json

```diff
@@ -6,12 +6,17 @@
   "module": "./dist/index.module.js",
   "scripts": {
     "build": "rollup -c ./scripts/build.mjs",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "pretest:e2e": "cd test/e2e && pnpm install",
+    "test:e2e": "pnpm --filter e2e-tests test"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/JoviDeCroock/GraphQLSP.git"
   },
+  "files": [
+    "dist"
+  ],
```

Comment:
Added this so `test` folder is not published

Reply:
Let's add it to npmignore instead, having one standard will reduce confusion